### ISSUE TITLE
WFS Layers not Refreshing

### ISSUE THREAD

**Issue by geographika** [compassinformatics/cpsi-mapview#203]
The `CpsiMapview.view.menuitem.LayerRefresh` class seems to fail to refresh a WFS layer - see [here](https://github.com/compassinformatics/cpsi-mapview/blob/5c98288a7f1b195ce55fb77e69a9d304dc50aa06/app/view/menuitem/LayerRefresh.js#L59)

A screencast of the issue is available at https://drive.google.com/file/d/1azO1N_ur1wQDRmDRsfEok_TFqw91YgWD/view (the Chrome tools network tab is hidden but no network request is made). 

The code at https://github.com/compassinformatics/cpsi-mapview/blob/master/app/controller/grid/Grid.js#L152 also seems to fail to refresh a WFS-based layer. I tried a few different approaches with no success:

```
            vectorSource.changed();
            //vectorSource.clear();
            //vectorSource.refresh();
```

**Comment by geographika** [comment]
Will need to investigate further as this works fine for some WFS.

**Comment by simonseyock** [comment]
Prior to openlayers version 6, the refresh method does not reload the source.

But the clear should clear the loaded extents (before version 6) and calling a refresh after that - should - then reload the source. If this does not work the problem lies somewhere else.

See https://github.com/openlayers/openlayers/commit/f40cbf2cac8b469388fd23e506f7bae15f69d3c6 for a little more information.

**Comment by geographika** [comment]
@simonseyock - I've added in:

```
source.clear();
source.refresh();
```

 to https://github.com/compassinformatics/cpsi-mapview/blob/2476b20d15d7071c00c18ee9850a67fa1fbbc3db/app/util/Layer.js#L30

There are still no network requests are made when refreshing at the following location (which has points): map=11/-995536.51/6987516.99/0

![image](https://user-images.githubusercontent.com/490840/94678034-28012d00-031e-11eb-9f29-875bdde024ca.png)

I thought it might be due to no features, but this does not seem to be the case. I added a debugger statement and tried various attempts in the console but can't seem to trigger a network request.

**Comment by simonseyock** [comment]
The borehole layer is clustered. This way source is not the original source, but the cluster source. You can circumvent that by adding
```
if (source.getSource) {
    source = source.getSource();
}
```
after line https://github.com/compassinformatics/cpsi-mapview/blob/2476b20d15d7071c00c18ee9850a67fa1fbbc3db/app/util/Layer.js#L21

**Comment by simonseyock** [comment]
and a `source.clear()` seems to be enough

**Comment by geographika** [comment]
Thanks @simonseyock - thought I must be missing something. Fixed with #283
